processing/src/main/java/io/druid/segment/incremental/IncrementalIndex.java

```diff
@@ -80,265 +64,10 @@
 
 /**
  */
-public abstract class IncrementalIndex<AggregatorType> implements Iterable<Row>, Closeable
+public abstract class IncrementalIndex<AggregatorType> extends IncrementalIndices implements Iterable<Row>, Closeable
```

Comment:
This seems odd to me too, it would make more sense to me if things were called statically like `IncrementalIndices.STRING_TRANSFORMER`